app/javascript/components/widgets/weather/current-weather.jsx

```diff
@@ -32,6 +33,8 @@ const CurrentWeather = ({ weather, useLargeIcon }) => {
         <SmallSkyIcon icon={icon} />
       )}
       {description}
+      {weather.current.windGust > 15 &&
+        ` , gusty wind at ${weather.current.windGust} mph`}
```

Comment:
Lets extract this into a component.

Nit: the extra space before the comma seems incorrect.